comet/utils/redis.py

```
@@ -0,0 +1,191 @@
+            return False
+        except Exception as e:
+            logger.error(f"Unexpected error during Redis SET for key {self._sanitize_key_for_logging(key)}: {e}")
+            return False
+
+    async def delete(self, key: str) -> bool:
+        if not self.is_connected():
+            return False
+        
+        try:
+            await self.redis.delete(key)
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)